### DISCUSSION TITLE
Welcome to Roboyard Discussions!

### DISCUSSION THREAD

**Discussion by rubo77** [Eastcoast-Laboratories/Roboyard#13] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by narratorjay** [comment]
So addictive!  And a great dopamine hit when I finally see how the moves link together for the right robot to reach the target.  **3D please!!**
In my interest in 3D work I create digital animations of real puzzles - https://github.com/narratorjay/j3-j

**Comment by rubo77** [comment]
3D for this game would be a real challenge! maybe suggest this to drifting droids: https://github.com/smack42/DriftingDroids the programmer there is a real genius ;)

**Reply by narratorjay** [reply]
Thank you rubo77.  Still enjoying the dopamine hit!  btw I'm playing it on LineageOS, downloaded Roboyard from F-Droid.
